dataland-csvconverter/src/main/kotlin/org/dataland/csvconverter/csv/EuTaxonomyUtils.kt

```diff
@@ -0,0 +1,62 @@
+package org.dataland.csvconverter.csv
+
+import org.dataland.csvconverter.csv.CsvUtils.getCsvValue
+import org.dataland.datalandbackend.model.enums.eutaxonomy.AttestationOptions
+import org.dataland.datalandbackend.model.enums.eutaxonomy.YesNo
+
+/**
+ * This objects provides parsing methods for columns that are required by both eu-taxonomy frameworks
+ * but don't belong to the companyInformation parser
+ */
+object EuTaxonomyUtils {
+
+    private const val REPORT_OBLIGATION_YES = "Yes"
+    private const val REPORT_OBLIGATION_NO = "No"
+    private const val REPORT_OBLIGATION_NA = "n/a"
+
+    private const val ATTESTATION_REASONABLE = "reasonable"
+    private const val ATTESTATION_LIMITED = "limited"
+    private const val ATTESTATION_NA = "n/a"
+    private const val ATTESTATION_NONE = "none"
+
+    private val columnMappingEuTaxonomyUtils = mapOf(
+        "reportObligation" to "NFRD mandatory",
+        "attestation" to "Assurance",
+    )
+
+    /**
+     * This function parses the reportingObligation field from the Eu-Taxonomy framework CSV file
+     */
+    fun getReportingObligation(csvLineData: Map<String, String>): YesNo {
+        val rawReportObligation = columnMappingEuTaxonomyUtils.getCsvValue("reportObligation", csvLineData)
```

Comment:
man könnte die Variablendefinition auch in das when packen (schlägt die IDE auch vor): return when (val rawReportObligation = columnMappingEuTaxonomyUtils.getCsvValue("reportObligation", csvLineData)) {

Reply:
Done